src/RedshiftBase.php

```diff
@@ -53,13 +53,20 @@ public function import($tableName, $columns, array $sourceData, array $options =
                 $tableName,
                 $primaryKey,
                 $columns,
-                $options['useTimestamp']
+                $options['useTimestamp'],
```

Comment:
nejsem si jistej jestli tu default hodnotu nakonec taky neosetrit tady. Protoze ted kdyz to nekdo zavola s 
```
[
 'nullify' => ['neco'],
]
```
tak to zarve na undefined index `useTimestamp`.